rpcserverhelp.go

```diff
@@ -717,6 +717,23 @@ var helpDescsEnUS = map[string]string{
 	"versionresult-patch":         "The patch component of the JSON-RPC API version",
 	"versionresult-prerelease":    "Prerelease info about the current build",
 	"versionresult-buildmetadata": "Metadata about the current build",
+
+	// TestMempoolAcceptCmd help.
+	"testmempoolaccept--synopsis":  "Returns result of mempool acceptance tests indicating if raw transaction(s) would be accepted by mempool.",
+	"testmempoolaccept-rawtxns":    "Serialized transactions to test.",
+	"testmempoolaccept-maxfeerate": "Maximum acceptable fee rate in BTC/kB",
+
+	// TestMempoolAcceptCmd result help.
+	"testmempoolacceptresult-txid":             "The transaction hash in hex.",
```

Comment:
Ok cool, I see this here, so can ignore that other comment about needing to update some other docs.